### DISCUSSION TITLE
Settings under username

### DISCUSSION THREAD

**Discussion by malckier** [entel-me/entel#74] [Ideas]
It'd be good to see the username that is currently logged in. Many sites include a dropdown near the user name to allow for settings to be changed. I noticed the email settings in the issues list and perhaps it could be included that way.

**Comment by antonykamp** [comment]
Hey malckier,
I like your idea! Let's think a bit bigger ;) What do you think about a "change user settings"-entry in the dropdown menu? You could see and change your username (, mail), notification settings, password, and more :D

**Reply by malckier** [reply]
Those sound like great ideas!

**Reply by antonykamp** [reply]
The user should be able to see
- name
- mail

and the user should be able to change
- name
- password
- notification setting.

Which notification-setting should we provide? 🤔 Mail on/off? (Push notifications on/off in the future?) Which message types she/he wants to receive....more?

**Reply by malckier** [reply]
I would think mail on/off for the notifications:
- Chat message received
- List accepted
- List completed
- Maybe a notification if a new list exists for someone who completed your list, or vice-versa.

**Reply by antonykamp** [reply]
> Maybe a notification if a new list exists for someone who completed your list, or vice-versa.

I really like this idea 🚀
